### ISSUE TITLE
AgoraLiveInjectStreamConfig audioSampleRate problem

### ISSUE THREAD

**Issue by emingli360** [AgoraIO-Community/Agora-Flutter-Quickstart#74]
Exception thrown when using `AgoraAudioSampleRateType` and will exit current function without running the next lines.
```dart
    await AgoraRtcEngine.addPublishStreamUrl(
        RTMP_LINK, false);
    await AgoraRtcEngine.addInjectStreamUrl(
        RTMP_LINK,
        AgoraLiveInjectStreamConfig.fromJson({
          "width": 720,
          "height": 1280,
          "videoGop": 30,
          "videoBitrate": 3420,
          "audioBitrate": 48,
          "audioSampleRate": AgoraAudioSampleRateType.MidRateType
        }));

    // Next line of code wont execute
```
Error message
``` dart
E/MethodChannel#agora_rtc_engine(32757): Failed to handle method call
E/MethodChannel#agora_rtc_engine(32757): java.lang.ArrayIndexOutOfBoundsException: length=3; index=44100
E/MethodChannel#agora_rtc_engine(32757): 	at io.agora.agorartcengine.AgoraRtcEnginePlugin.onMethodCall(AgoraRtcEnginePlugin.java:614)
E/MethodChannel#agora_rtc_engine(32757): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:231)
E/MethodChannel#agora_rtc_engine(32757): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:93)
E/MethodChannel#agora_rtc_engine(32757): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:642)
E/MethodChannel#agora_rtc_engine(32757): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/MethodChannel#agora_rtc_engine(32757): 	at android.os.MessageQueue.next(MessageQueue.java:336)
E/MethodChannel#agora_rtc_engine(32757): 	at android.os.Looper.loop(Looper.java:174)
E/MethodChannel#agora_rtc_engine(32757): 	at android.app.ActivityThread.main(ActivityThread.java:7682)
E/MethodChannel#agora_rtc_engine(32757): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#agora_rtc_engine(32757): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
E/MethodChannel#agora_rtc_engine(32757): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
```

**Comment by LichKing-2234** [comment]
pls use 0 to 3 for `audioSampleRate`, we will fix it later

**Comment by LichKing-2234** [comment]
fixed in 1.0.11
